verifier.yml

```
@@ -62,11 +62,12 @@ paths:
         description: Parameters for verifying a verifiablePresentation.
       responses:
         "200":
-          description: Verifiable Presentation successfully verified!
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/VerifyPresentationResponse"
+            description: Verifiable Presentation successfully verified!
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/VerifyPresentationResponse"
+
```

Again, seems like an accidental change to proper 2-space indentation.

```suggestion
          description: Verifiable Presentation successfully verified!
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/VerifyPresentationResponse"

```